### ISSUE TITLE
[TypeScript] Found 0 errors. Watching for file changes.

### ISSUE THREAD

**Issue by jaychoubaby** [abi/screenshot-to-code#165]
I have deployed it using Docker, but when accessing the backend service, there is a prompt.

`Your backend is running correctly. Please open the front-end URL (default is http://localhost:5173) to use screenshot-to-code.`

Error accessing front-end service as follows：`[TypeScript] Found 0 errors. Watching for file changes.`

**Comment by limboinf** [comment]
This looks very confusing. It should be an issue with the lasted version of the vite-plugin-checker plugin:
https://github.com/fi3ework/vite-plugin-checker/issues/238

**Comment by abi** [comment]
Can you open up your developer console and show me the logs? 

Or you can use the hosted version at screenshot-to-code.com

**Comment by abi** [comment]
That didn't really help. I think the issue is most likely that your front end isn't connecting to the backend. 

127.0.0.1:7001 opens fine on your browser?

are you running this locally?

**Comment by jaychoubaby** [comment]
> Ah that's the problem then.
> 
> The problem is when you access the IP from your local machine, the backend IP you need to use can't be the internal machine IP, it has to be the external IP.
> 
> So updated frontend/.env.local:
> 
> VITE_WS_BACKEND_URL=ws://..

I have added it, but it seems to have no effect and still points to 127.0.0.1. I deployed using Docker, does this have anything to do with it?

**Comment by abi** [comment]
Yeah if you update it, you need to re build the docker image.

**Comment by jaychoubaby** [comment]
> Yeah if you update it, you need to re build the docker image.

Yes, I have made configuration changes and redeployed the image, but the same error persists.
![image](https://github.com/abi/screenshot-to-code/assets/26641387/1da904bf-28bc-4ef8-8a29-294fbae594cd)

**Comment by abi** [comment]
That .env is in frontend, not backend

**Comment by jaychoubaby** [comment]
> That .env is in frontend, not backend
Now it's okay, thank you very much for your help.
